src/DynamoCore/Properties/Resources.resx

```diff
@@ -503,7 +502,7 @@ If you don't mind, it would be helpful for you to send us your file. That will m
   </data>
   <data name="ToolTipSymbol" xml:space="preserve">
     <value>Symbol</value>
-	</data>
+  </data>
```

Comment:
`Resources.resx` should be exactly the same as `Resources.en-US.resx`, can you copy `Resources.en-US.resx` and overwrite `Resources.resx` since `Resources.resx` has some empty line changes?